### ISSUE TITLE
.NET Standard class library fails to build (VS 2017 image)

### ISSUE THREAD

**Issue by discosultan** [appveyor/ci#1381]
I have a solution with mixed .NET Standard (1.3) and .NET Framework (4.6.1) class libraries. I have created a new AppVeyor project directly from [GitHub project](https://github.com/discosultan/VulkanCore) and simply changed the environment worker build image from VS 2015 to VS 2017. I wonder if I'm missing something here ...

The build failure starts with:

```
C:\Users\appveyor\AppData\Local\Temp\1\.NETStandard,Version=v1.3.AssemblyAttributes.cs(4,20): error CS0400: The type or namespace name 'System' could not be found in the global namespace (are you missing an assembly reference?)
```

Running `msbuild` on the solution works fine locally.

**Comment by IlyaFinkelshteyn** [comment]
Try to run `nuget restore` as **before build** or **install** script

**Comment by discosultan** [comment]
@IlyaFinkelshteyn That makes sense, thanks!

I'm now stuck at a next error:

```
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\Microsoft\NuGet\15.0\Microsoft.NuGet.targets(197,5): error : The project.json is referencing the project 'C:\projects\vulkancore\Tests\Src\VulkanCore.csproj', but an output path was not specified on an item in the ProjectReferencesCreatingPackages property.
```

**Comment by FeodorFitsner** [comment]
How do you currently build the solution from command line? I've logged into a build worker via RDP and was able to build the solution from VS 2017 IDE. But I couldn't built it from a command line.

**Comment by discosultan** [comment]
@FeodorFitsner Thanks for your time! Indeed, `msbuild` only worked for me because I did not do a clean install - I had intermediary files already populated through an IDE build.

I still haven't figured out the reason of failure, though. I managed to build the solution from cmd line by building with `devenv` **twice** (first build fails with a similar error as `msbuild`). I'll keep investigating ...
